docs/review_process.md

```diff
@@ -0,0 +1,148 @@
+
+- [@madebr](https://github.com/madebr)
+- [@SpaceIm](https://github.com/SpaceIm)
+- [@ericLemanissier](https://github.com/ericLemanissier)
+- [@prince-chrismc](https://github.com/prince-chrismc)
+- [@Croydon](https://github.com/Croydon)
+- [@intelligide](https://github.com/intelligide)
+- [@theirix](https://github.com/theirix)
+- [@gocarlos](https://github.com/gocarlos)
+
+The list is not constant and will change periodically based on contribution.
+That also means you can be included in this list as well - submit PRs and provide reviews, and in time you may be added as a trusted contributor.
+
+### Rule of 3 reviews
+
+At least 3 reviews are required for an approval, and at least one of them has to be from the official reviewers.
+So, it might be 1 official + 2 community, or 3 official, but it couldn't be just 3 community reviews.
+
+### Reviews from others
+
+All reviews are still valuable and very helpful. Even if you're not listed as an official or community reviewer counted by the bot, your reviews are still welcome, so please do not hesitate to provide them.
+
+### Addressing review comments
+
+Please ensure to address the review comments and respond to them in order to get your PR approved and finally merged.
+It doesn't always mean accepting all the suggestions, but at least providing a response, so people can understand your position.
+
+## Automatic Merges
+
+The bot runs Automatic Merges every 30 minutes. Currently, it can only merge a single PR in this timeframe, so there is a theoretical limit of 48 PRs merged per day (in practice, it's even less for reasons listed below).
+PR is selected for the merge only if:
+
+- author is a [beta user](https://github.com/conan-io/conan-center-index/issues)
+- author has signed CLA
+- PR is not a Draft or WIP
+- PR has a green status (successful build)
+- PR doesn't have merge conflicts with `master` branch
+- PR has 3 approved reviews (rule described above)
+- master build is not running already (see below)
+
+If these conditions are fullfilled, the PR is merged (associated issues are automatically closed), and then build of master is launched
+
+### Merge
+
+If there is an actual merge to happen (for instance, the recipe changed in PR was already updated in master by the time PR merged), 
+it will introduce a new recipe revision. Therefore, the build should be run one more time, so the master build is launched.
+In reality this could happen frequently enough if there are multiple PRs aiming to update the same recipe (even if they touch different files in the same recipe).
+Such builds can takes hours for big packages (like boost), blocking other merges for a while.
+
+### Upload
+
+Even if there is no new revision introduced, CI still needs to publish artifacts from the internal repository to the conan center index, and it also currently may take hours for big enough packages.
```

Comment:
```suggestion
Even if there is no new revision introduced, CI still needs to publish artifacts from the internal repository to ConanCenter, and it may also take hours for big enough packages.
```